### PR TITLE
New Template: Cisco ASA: 'show cpu usage detailed'

### PR DESCRIPTION
**Pull request by yone2ks**
##### ISSUE TYPE
 - New Template Pull Request

##### COMPONENT
cisco_asa_show_cpu_usage_detailed.textfsm, .*, cisco_asa, sh[[ow]] cpu u[[sage]] d[[etailed]]

##### SUMMARY
Adding template for Cisco ASA for the command "show cpu usage derailed"

### CODE REVIEW

File: ntc_templates/templates/cisco_asa_show_cpu_usage_detailed.textfsm [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Value CPU_1_MIN (\d+\.\d+)
+Value CPU_1_MIN_DP (\d+\.\d+)
+Value CPU_1_MIN_CP (\d+\.\d+)
+Value CPU_5_MIN (\d+\.\d+)
+Value CPU_5_MIN_DP (\d+\.\d+)
+Value CPU_5_MIN_CP (\d+\.\d+)
+
+
+Start
+  ^${CORE}\s+${CPU_5_SEC}\s+\(${CPU_5_SEC_DP}\s+\+\s+${CPU_5_SEC_CP}\)\s+${CPU_1_MIN}\s+\(${CPU_1_MIN_DP}\s+\+\s+${CPU_1_MIN_CP}\)\s+${CPU_5_MIN}\s+\(${CPU_5_MIN_DP}\s+\+\s+${CPU_5_MIN_CP}.+ -> Record
```
Comment (jvanderaa) [inline ntc_templates/templates/cisco_asa_show_cpu_usage_detailed.textfsm:14]: Please add the error handling as well. Such as https://github.com/networktocode/ntc-templates/blob/master/ntc_templates/templates/alcatel_aos_show_vlan.textfsm#L18

**Review by jvanderaa — CHANGES_REQUESTED** [review]

File: ntc_templates/templates/cisco_asa_show_cpu_usage_detailed.textfsm (hunk 1 above)
Comment (yone2ks) [reply ntc_templates/templates/cisco_asa_show_cpu_usage_detailed.textfsm]: Thank you for your review. I added the error handling.
Please review it.

**Comment by yone2ks** [comment]
i have a build error only python 3.7.  I don't know what caused it.
Please a advice for me.

**Review by jvanderaa — APPROVED** [review]

**Comment by jvanderaa** [comment]
Thanks @yone2ks!

**Pull request merged**